Formula/stolon.rb

```diff
@@ -0,0 +1,33 @@
+$version = "v0.13.0"
+$bin_name = "stolon"
+
+class Stolon < Formula
+    desc "Stolon is a cloud native PostgreSQL manager for high availability"
+    homepage "https://github.com/sorintlab/stolon"
+    url "https://github.com/sorintlab/#{$bin_name}.git", tag: $version
+    version $version
+
+    depends_on "go" => :recommended
```

Comment:
Don't use recommended. It doesn't build without go.

Reply:
we made it `build`